### ISSUE TITLE
Multipath support not enabled in kernel

### ISSUE THREAD

**Issue by omkhar** [clearlinux/distribution#2268]
Please enable multipath support in the Clear Linux Kernel

```
$ ip route add default scope global nexthop via 1.1.1.1 dev enp1s0 weight 1 nexthop via 2.2.2.2 dev enp2s0 weight 1
Error: Multipath support not enabled in kernel.
```

```
$ uname -r
5.10.10-1017.native
```

**Comment by mdhorn** [comment]
Config was updated. Should be in the next Clear Linux OS release.

**Comment by omkhar** [comment]
ack confirmed fixed in 5.10.12-1021.native
